devstack/lib/nfp

```diff
@@ -123,10 +123,6 @@ function create_ep_and_nsp {
  --shared True\
  default
 
-    gbp ep-create\
- --external-segments default\
- ext_connect
```

Comment:
if not then one should add in the other scripts as well to test for North-South traffic.